lib/Net/DRI/DRD/Afilias/Shared.pm

```diff
@@ -141,4 +141,14 @@ sub verify_name_domain
 }
 
 ####################################################################################################
+####################################################################################################
```

Comment:
You seem to have moved PRO into Donuts, but the Pro Extension functions into Afilias::Shared. Can you clarify what's happening here? Is the PRO extension not supported in Donuts? In which case, is this functionality deprecated completely?

Reply:
According to Paulo Jorge, we moved the Pro to Shared and would cleanup at a later date after discussion. Donuts doesn't support the Pro extension.